### PR TITLE
Add TA and initial schedule

### PR DESCRIPTION
**Pull request by rgiordan**

### CODE REVIEW

**Comment by rgiordan** [comment]
@ryanlovett Here is an initial PR.

**Comment by ryanlovett** [comment]
I just sent you an invite to the spring-2024-instructors team which has the role of Maintainer.

Comments to follow...

**Review by ryanlovett — APPROVED** [review]
Looks great. The images of folks not associated with Stat 151A can be removed.

Once you accept the new role invite, you'll be able to self merge!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # stat999-myst
 
+# https://stat151a.stat.berkeley.edu/spring-2024/
```
Comment (ryanlovett) [inline README.md:3]: Though I believe this was intended as a hidden bookmark, the `#` actually declares that the URL should be formatted as a level 1 heading.

File: index.md [hunk 2]
```diff
@@ -1,36 +1,59 @@
...
+  <div>
+    <h3 class="staffer-name">
+      <a href="{{ staff_website }}" target="_blank">{{ staff_name }}</a>
+      <p class="staffer-pronouns"><b>{{ staff_pronouns }}</b></p>
+    </h3>
+    <p><a href="mailto:{{ staff_email }}">{{ staff_email }}</a></p>
+    <p><b>Office Hours:</b> {{ staff_oh }}</p>
+  </div>
+</div-->
+
```
[the remark this answers is not in the document]
Comment (rgiordan) [reply index.md]: Done

**Pull request merged**